packages/ember/src/components/footer.gts

```diff
@@ -0,0 +1,32 @@
+import type { TOC } from '@ember/component/template-only';
+
+interface FooterSignature {
+  Element: HTMLElement;
+  Blocks: {
+    left?: [];
+    right?: [];
+    default?: [];
+  };
+}
+
+const Footer: TOC<FooterSignature> = <template>
+  <footer
+    class="bg-body-secondary border-top border-2 border-dark-subtle mt-auto"
+    ...attributes
+  >
+    <div class="container-fluid py-2 px-4">
+      <div
+        class="row row-cols-auto justify-content-between align-content-center"
+      >
+        <div class="col row row-cols-auto align-content-center">
+          {{yield to="left"}}
+        </div>
+        <div class="col row row-cols-auto align-content-center">
+          {{yield to="right"}}
+        </div>
```

Comment:
This looks good to me, but on mobile it does break to a new line. I know the design files show the mobile view as one line, but I'm not sure how we could work that out and still fit all of the content

Reply:
The latest commit makes the text smaller on all screen sizes. Here is what it looks like on the iPhone XR
![image](https://github.com/user-attachments/assets/2080f075-8612-4ae7-9777-053d0aad4b93)

But it still break into a new line on the iPhone SE
![image](https://github.com/user-attachments/assets/b55bd8a1-d619-4b58-aa51-6a5c9cc8bf83)